biogrid/libs/biogrid-simulator/src/lib/biobattery/biobattery-class.ts

```diff
@@ -86,7 +86,7 @@ export class BioBattery implements Battery {
     return validate(batteryValidator);
   }
 
-  getEnergyInJoules(): Energy {
+  getEnergyInKilowattHour(): Energy {
```

Comment:
could you also change the variable name to `energyInKiloWattHour`

Reply:
+1 variable name and getter name should match